product/reports/670_Performance by Asset Type - Middleware Servers/160_Web Metrics - hourly over last day.yaml

```diff
@@ -0,0 +1,59 @@
+---
+where_clause:
+dims:
+reserved:
+title: "EAP Web Metrics - hourly over the last day"
+conditions:
+order: Descending
+graph:
+menu_name: "EAP Web Metrics - hourly over the last day"
+rpt_group: Custom
+priority:
+col_order:
+- id
+- name
+- start_time
+- mw_aggregated_active_web_sessions_max
+- mw_aggregated_max_active_web_sessions_max
+- mw_aggregated_expired_web_sessions_max
+- mw_aggregated_rejected_web_sessions_max
+- mw_aggregated_servlet_time_avg
+- mw_aggregated_servlet_request_count_max
+timeline:
+file_mtime:
+categories:
+rpt_type: Custom
+filename:
+db_options:
+  :rpt_type: middleware
+  :options:
+    :interval: hourly
+    :start_offset: 86400
+    :end_offset: 0
+include: {}
+
+db: MiddlewareServerPerformance
+cols:
+- id
+- name
+- start_time
+- mw_aggregated_active_web_sessions_max
+- mw_aggregated_max_active_web_sessions_max
+- mw_aggregated_expired_web_sessions_max
+- mw_aggregated_rejected_web_sessions_max
+- mw_aggregated_servlet_time_avg
+- mw_aggregated_servlet_request_count_max
+template_type: report
+group: n
+sortby:
+- id
+headers:
+- Id
+- Name
+- Start
+- Aggregated Active Web Sessions
+- Aggregated Max Active Web Sessions
+- Aggregated Expired Web Sessions
+- Aggregated Rejected Web Sessions
+- Aggregated Servlet Request Time
```

Comment:
Probably, _Request Time_ should be labeled as _Average_ while the others as _Maximum_ values?
Also, the word _Aggregated_ doesn't make sense to me in the columns.

Reply:
+1 to remove Aggregated word to label.
+1 to add Average to Servlet Request Time
For the other labels, I do have doubts about adding a Maximum label. Those values are counters of sessions (created, active, rejected, so on) so maximum doesn't add meaning.

Reply:
Probably, it's worth only for the `Active Web Session` column since data doesn't look like a proper counter. May be this one is the observed highest number of active web sessions? If so, it's like a gauge rather than a counter. (If it were a counter, the label should instead be `Handled Web Sessions`).

For all others columns (and excluding _Servlet Request Time_) ... I cannot understand why in your updated image there are fractional numbers. These counters shouldn't have fractional numbers. This alone may make the user unsure about what kind of numbers is the report outputting.